content/tutorials/creating-resources-gcp/index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Creating Resources on Google Cloud"
+title_tag: "Creating Resources on Google Cloud"
+layout: single
+
+# A succinct description of the tutorial. It appears on the Tutorials home and collection pages.
+description: Learn how to define and provision resources on Google Cloud using Pulumi.
+
+# A similar description used for search results and social-media previews.
+meta_desc: Learn how to define and provision resources on Google Cloud using Pulumi.
+
+# An image for the tutorial. It appears on tutorial page and in social-media previews.
+meta_image: create-resources-gcp-meta.png
+
+# An optional video for the tutorial. When present, it appears at the top of the page, replacing
+# the meta image. YouTube and HTML5 video sources are supported.
+# video:
+#     url: /blog/drift-detection/drift.mp4
+#     youtube: Q8tw6YTD3ac
+
+# The order in which the tutorial appears in most lists. Order is ascending, so higher numbers
+# mean the tutorial will appear further down the list. Positive integers only.
+weight: 999
+
+# A brief summary of the tutorial. It appears at the top of the tutorial page. Markdown is fine.
+summary: |
+   In Pulumi, resources represent the fundamental units that make up your infrastructure, such as virtual machines, networks, storage, and databases. A resource is used to define and manage an infrastructure object in your Pulumi configuration.
+   In this tutorial, you will create a simple static website hosted on an Azure Blob Storage account. You will then refer to documentation in the Pulumi Registry to configure the storage account as a website.
```

Comment:
"...hosted on an Azure Blob Storage account."=>"...hosted on a Google Cloud virtual machine."

Reply:
Updated, and yes the plan is to replace the Learn content with the updated newer tutorials.